packages/autocomplete/src/examples/multiselect.md

```diff
@@ -0,0 +1,304 @@
+                      small: true,
+                      select: true,
+                      tagLayout: true,
+                      focused: isOpen || typeof tagFocusedKey !== 'undefined' || state.isFocused,
+                      inputRef: ref => {
+                        this.wrapperRef = ref;
+                        triggerRef(ref);
+                      }
+                    })}
+                  >
+                    {renderTags(
+                      isOpen || typeof tagFocusedKey !== 'undefined' || state.isFocused,
+                      state.selectedKeys,
+                      tagFocusedKey,
+                      getTagProps
+                    )}
+
+                    <Input
+                      {...getInputProps(
+                        getFieldInputProps({
+                          bare: true,
+                          innerRef: inputRef,
+                          value: state.inputValue,
+                          onChange: e => {
+                            setState({ inputValue: e.target.value });
+                          },
+                          onKeyDown: e => {
+                            if (tagFocusedKey !== undefined) {
+                              if (
+                                e.keyCode === KEY_CODES.DELETE ||
+                                e.keyCode === KEY_CODES.BACKSPACE
+                              ) {
+                                deleteTag(tagFocusedKey);
+                              }
+
+                              // copy-to-clipboard functionality
+                              if (e.keyCode === 67 && e.metaKey) {
+                                alert(`"${tagFocusedKey}" copied`);
+                              }
+                            }
+                          },
+                          placeholder:
+                            Object.keys(state.selectedKeys).length === 0
+                              ? 'Enter some content'
+                              : undefined,
+                          onFocus: () => {
+                            setState({ isFocused: true });
+                          },
+                          onBlur: () => {
+                            setState({ isFocused: false });
+                          },
+                          style: Object.assign(
+                            { marginLeft: '2px', flexGrow: 1, width: 60 },
+                            Object.keys(state.selectedKeys).length !== 0 &&
+                            (!state.isFocused || tagFocusedKey !== undefined)
+                              ? { opacity: 0, height: 0, minHeight: 0 }
+                              : {}
+                          )
+                        })
+                      )}
+                    />
+                  </FauxInput>
+                );
+              }}
+            >
+              {({ getMenuProps, getItemProps, placement, focusedKey }) => {
+                return (
+                  <MenuView
```

Comment:
Something is wrong with the initial top placement of this menu.

<img width="526" alt="screen shot 2018-06-29 at 3 35 17 pm" src="https://user-images.githubusercontent.com/143773/42117560-26eb3280-7bb2-11e8-85cf-372061a47965.png">

Reply:
This is currently being address in the SplitButton PR https://github.com/zendeskgarden/react-components/pull/48

Looks like a regression happened in how chrome/firefox renders `translatex` placements with our animations nested within it. No version changes have occurred in popper.js or react-popper

Reply:
Ok. Will look at that PR for the fix – we also lost animation on popping the menu up (it's still there on down).